build.sbt

```diff
@@ -118,6 +118,10 @@ lazy val xjcSettings =
     xjcCommandLine += "-p",
     xjcCommandLine += "org.apache.daffodil.tdml",
     xjcBindings += "debugger/src/main/resources/bindings.xjb",
+    xjcJvmOpts ++= Seq(
+      // workaround: certain reflection (used by JAXB) isn't allowed by default in JDK 17: https://docs.oracle.com/en/java/javase/17/migrate/migrating-jdk-8-later-jdk-releases.html#GUID-7BB28E4D-99B3-4078-BDC4-FC24180CE82B
```

Comment:
Just break the too-long comment lines here and below.

Reply:
The URL is very long, but I'll make the rest break better.